documentation/tutorials/Demo_1_Basic_Run/pecan.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<pecan>
+ <info>
+  <notes/>
+  <userid>-1</userid>
+  <username/>
+  <date>2025-06-19-15-34-01</date>
+ </info>
+ <outdir>demo_outdir</outdir>
+ <pfts>
+  <pft>
+   <name>temperate.coniferous</name>
+   <posterior.files>pft/temperate.coniferous/prior.distns.Rdata</posterior.files>
+   <outdir>pft/temperate.coniferous/pft/temperate.coniferous</outdir>
+  </pft>
+ </pfts> 
+ <ensemble>
+   <size>1</size>
+   <variable>NPP</variable>
+   <samplingspace>
+   <parameters>
+    <method>uniform</method>
+   </parameters>
+   </samplingspace>
+  </ensemble>
```

Comment:
To address an earlier comment about not introducing the ensemble concept yet, would this be sufficient for the demo?

```suggestion
 <ensemble>
   <variable>NPP</variable>
  </ensemble>
```